src/ng/http.js

```diff
@@ -909,7 +914,7 @@ function $HttpProvider() {
   var data = element(by.binding('data'));
   var fetchBtn = element(by.id('fetchbtn'));
   var sampleGetBtn = element(by.id('samplegetbtn'));
-  var sampleJsonpBtn = element(by.id('samplejsonpbtn'));
+//  var sampleJsonpBtn = element(by.id('samplejsonpbtn'));
```

Comment:
It's unused because the test that was using it was commented out and ESLint was shouting about an unused variable.